requirements.in

```diff
@@ -35,7 +35,7 @@ pytest-dotenv
 pytest-env==0.6.2
 pytest==6.2.5
 python-dotenv
-sentry-sdk
+sentry-sdk==1.11.1
```

Comment:
What's the need to pin this?

Reply:
@marksweb Without pinning it, running `pip-compile` would result in the library keeping the old version. The commit with this change is in another PR and I based off this branch instead of `main` by mistake. Will rebase later.

Reply:
Rebased and dropped that commit with the pillow and sentry-sdk upgrades